doc/rst/technotes/libraries.rst

```diff
@@ -551,6 +558,149 @@ include the chapel runtime and standard modules for use in a non-Chapel program
 and when the library is linked to a Chapel program this leads to multiple
 definitions of these functions.
 
+Using Your Library in Multilocale Settings
+==========================================
+
+Prerequisites
+-------------
+
+Chapel supports ``--library`` when ``CHPL_COMM = gasnet``.  We intend to support
+other settings in the future, see :ref:`Other Settings` in the :ref:`Multilocale
+Caveats` section for more information.
+
+To compile a multilocale library, `ZeroMQ <http://zeromq.org/>`_ must be
+installed.
+
+If ZeroMQ is not installed in a way that enables your C compiler to find it
```

Comment:
>  in a way that enables your C compiler to find it

Are there a specific set of env vars we should list as an example, such as through `LD_FLAGS` or `C_FLAGS` (assuming that's one way the C compiler could find ZMQ)

Reply:
I think `LD_LIBRARY_PATH`?  I worry about being too explicit about a strategy since there's such variation around installation across platforms

Reply:
Would listing a known working example such as `LD_LIBRARY_PATH` be problematic? I think that'd be sufficient for me.

Example text:

>...  in a way that enables your C compiler to find it, such as through the `LD_LIBRARY_PATH` environment variable, ...

Reply:
Honestly, given the amount of time I spent wrestling with finding libraries appropriately this summer, I would be worried about recommending a strategy that worked in some situations and not others instead of just letting them look it up for their particular situation

Reply:
Alright, sounds reasonable to play it safe then.